src/bin/stratis_dumpmetadata.rs

```diff
@@ -0,0 +1,47 @@
+use std::{
+    env,
+    fs::OpenOptions,
+    io::{Seek, SeekFrom},
+    process,
+    vec::Vec,
+};
+
+use serde_json::Value;
+
+use libstratis::engine::BDA;
+
+fn run(devpath: String) -> Result<(), String> {
+    let mut devfile = OpenOptions::new()
+        .read(true)
+        .open(&devpath)
+        .map_err(|_the_io_error| format!("Error opening device: {}", _the_io_error))?;
```

Comment:
Go ahead and remove the ```_``` from ```_the_io_error``` since you are now using it.

Reply:
Resolved.